internal/commands/server/k8s_e2e_test.go

```diff
@@ -32,8 +32,9 @@ import (
 )
 
 var (
-	testenv   env.Environment
-	hostRoute string
+	testenv      env.Environment
+	hostRoute    string
+	timeoutCheck = 1 * time.Minute
```

Comment:
bumped this up since, for some reason the enterprise version of consul, at least locally, seems to take longer registering services and pushing down xDS, which causes the timeouts in our `require.Eventually` expectations to get hit